EIPS/eip-3135.md

```diff
@@ -0,0 +1,203 @@
+
+| Parameter | Description |
+| ---------|-------------|
+| to | the deposit account |
+| amount | amount to return |
+
+#### depositBalanceOf
+
+Returns deposit balance and epoch of user account. The epoch is an increasing number which is added by one when account get claimed or withdraw. Epoch is a part of sign data, verified by both user and provider, prevents double spend and double claim.
+
+| Parameter | Description |
+| ---------|-------------|
+| user | user account address |
+
+#### claim
+
+Service provider (issuer) recovers unsigned message, then checks the message with it's signature (probably using built-in function 'ecrecover'). If the signature is successfully verified, provider get "credit" amount token from user's deposit balance and epoch is increased by 1. It is worth mentioning that the provider may not need to get token back because this token is basically created by this provider and may holds no value to itself. In that case, the claim function can release some other more valuable token to provider. 
+
+| Parameter | Description |
+| ---------|-------------|
+| from | which user account to be claimed |
+| credit | how many token should be claimed |
+| epoch | current epoch number |
+| signature| micropayment message signature signed by user|
+
+
+### Events
+
+#### Deposit
+
+Event emits when user calls deposit function
+
+| Parameter | Description |
+| ---------|-------------|
+| from | indexed msg.sender |
+| amount | deposit amount|
+
+#### Withdraw
+
+| Parameter | Description |
+| ---------|-------------|
+| to | the deposit account |
+| amount | amount to return |
+
+#### TransferIssuer
+
+Event emits when transferIssuer is called
+
+| Parameter | Description |
+| ---------|-------------|
+| oldIssuer | old issuer/beneficiary |
+| newIssuer | new issuer/beneficiary |
+
+#### Claim
+
+Event emits when claim is called
+
+| Parameter | Description |
+| ---------|-------------|
+| from | user account to be claimed |
+| to | msg.sender, the issuer |
+| epoch | which epoch is based on |
+| credit | how many token be claimed |
+
+## Rationale
+This standards provides a way for user and service provider to establish micropayment channel to make free and repaid payment messages off blockchain. Those payment messages are linked one by one, providing a trace of service consumption and keeps an increasing number of credit, which is how much the payer owned to the service provider.
+
+The provider can stop serving when payer no longer send new valid payment message or the credit is bigger than deposit balance. The user can stop signing new message if provider is not available or is cheating. The provider can claim token on chain at any time using the message contains the largest credit number.
+
+Let's explain the usage of exclusive claimable token by two cases.
```

Comment:
This seems to be more appropriate for the abstract.  The Rationale section should focus on specific choices this spec made that either do not have obvious reasons or may have alternative options available.  For example, "We chose constant X because ..." or "We went with the function name Y because ...".